_installation/mac_os.md

```diff
@@ -0,0 +1,118 @@
+
+{% highlight bash %}
+  > gcc --version
+{% endhighlight %}
+
+This should result in something similar to the following:
+
+{% highlight bash %}
+  Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
+  Apple LLVM version 8.0.0 (clang-800.0.38)
+  Target: x86_64-apple-darwin15.6.0
+  Thread model: posix
+  InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
+{% endhighlight %}
+
+
+## "Easy" ruby installs with rvm
+
+The [Ruby Version Manager (rvm)](http://rvm.io/) will allow you to install different versions of ruby on your machine and switch between them as you switch from one job to another.  This can save you from the pain of upgrading an old project to the newest ruby version when it's running along just fine.
+
+To install rvm, open a terminal window and run the following commands (copied directly off of the rvm website):
+
+{% highlight bash %}
+  > gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+\curl -sSL https://get.rvm.io | bash -s stable
+{% endhighlight %}
+
+Now that you have rvm installed, you can use it to install ruby.  By default, it installs the latest version, but if you want/need an older version, that's easy too.  For now, let's install the latest version (as of 12/3/2016), v2.3.1:
+
+{% highlight bash %}
+  > rvm install 2.3.1
+  > rvm default use 2.3.1
+{% endhighlight %}
+
+At this point, you should have ruby installed and available to you.
+
+{% highlight bash %}
+  > ruby -v
+  # => ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
+{% endhighlight %}
+
+## Install Rails
+
+Now that ruby is installed, installing rails is simple.  Just run the following command:
+
+{% highlight bash %}
+  > gem install rails 5.0.0
+{% endhighlight %}
+
+If that succeeds, you should be able to run rails on the command-line:
+
+{% highlight bash %}
+  > rails --version
+{% endhighlight %}
+
+## Test it out
+
+Now that we have everything we need installed, let's try it out.
+
+{% highlight bash %}
+  > rails new test_rails
+  > cd test_rails
+  > rails server
+{% endhighlight %}
+
+You should see output similar to this:
+
+{% highlight bash %}
+  => Booting Puma
+  => Rails 5.0.0.1 application starting in development on http://localhost:3000
+  => Run `rails server -h` for more startup options
+  Puma starting in single mode...
+  * Version 3.6.2 (ruby 2.3.1-p112), codename: Sleepy Sunday Serenity
+  * Min threads: 5, max threads: 5
+  * Environment: development
+  * Listening on tcp://localhost:3000
+  Use Ctrl-C to stop
+{% endhighlight %}
+
+If you see the output above, rails is running and you can point your browser at http://localhost:3000 and you should see this:
+
+{% include figure.html url="/assets/images/youre_on_rails.png" description="Yay! You're on Rails!" %}
```

Comment:
This is a good idea for dealing with images, but could we use the native Markdown syntax instead?

```
![Alt text](/path/to/img.jpg "Optional title")
```

https://daringfireball.net/projects/markdown/syntax#img